.github/actions/free-disk-space/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Is this github action still needed? If so I don't think we should bring in some external action for this but instead work on sharing artifacts instead of producing new ones.

Reply:
Yeah I removed it, it was an attempt to reduce disk space usage, but I introduced a bug that had caused that disk usage and fixing it solved the problem